### DISCUSSION TITLE
cannot find zipfile directory in one of /tmp/frontend.zip

### DISCUSSION THREAD

**Discussion by NAGENDRA5658** [learndevopsonline/learndevopsonline#1561] [Q&A]
when i am trying to execute the command ** unzip /tmp/frontend.zip**  getting an error as  cannot find zipfile directory in one of /tmp/frontend.zip
![dev-1](https://github.com/learndevopsonline/learndevopsonline/assets/98450158/30c79684-815f-4262-b381-ea561d8a8292)

**Comment by sreedharm07** [comment]
hi @NAGENDRA5658  you downloaded wrongly. it suppose to be **artifacts**. **s** is missing.
download properly and try it will work
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/dea2771b-efa0-41ae-a8e8-5377fe577dd7)

**Reply by NAGENDRA5658** [reply]
Thanks it worked
